sip-quickstart/.env

```diff
@@ -1,3 +1,14 @@
+# description: The name of your SIP application
+# format: text
+# required: true
 APP_NAME=SIP Quickstart
+
+# description: A password to protect your admin interface
+# format: secret
```

Comment:
for `secrets` that have a default value like these two, how are we proposing to treat this on a UI level? In a sense a secret would connote that the input field should be filled with *** values, but if we prepopulate the UI with the default, that's all they would see

Reply:
in other `secrets` there's no default value, and i wonder if we should keep it consistent with that approach

Reply:
I'm fine with removing it but we could also set the "default" as placeholder for secrets. Or choose to ignore the default and just require it.

Reply:
+1 to Shanan. would rather have users see that they need to fill this out in the UI.

Reply:
i feel that adds a lot of burden on the UI to edge case this behavior. Would rather see this more uniformly implemented